cli.js

```diff
@@ -43,12 +45,24 @@ function error(err) {
 function test(data) {
 	var isError = data.err.message;
 
+	var prefix = '';
+
+	if (fileCount > 1) {
+		var separator = ' ' + figures.pointerSmall + ' ';
+
+		prefix = data.file
+			.replace(process.cwd() + '/', '')
+			.replace(/test\-|test\//g, '')
```

Comment:
If the directory has `test-` then that's a problem. If they're on windows, `\/` might be a problem.